src/test/resources/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning_2_barn.feature

```diff
@@ -0,0 +1,35 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Beregning barnetilsyn 2 barn
```

Comment:
Burde vi ha en test for makssats flere barn? Siden makssatsen avhenger av antall barn (tror jeg)

Reply:
Det kan vi godt ha :)